geoserver_init/010_init-gs.js

```diff
@@ -96,13 +99,16 @@ async function createWorkspaces() {
 async function createPostgisDatastore() {
   framedMediumLogging('Creating PostGIS data store...');
 
-  const success = await grc.datastores.createPostgisStore(
+  const created = await grc.datastores.createPostgisStore(
     stationWorkspace, stationDataStore, pgHost, pgPort, pgUser, pgPassword,
     pgSchema, pgDb
   );
 
-  if (success) {
-    console.info('Successfully created PostGIS store');
+  if (created) {
+    console.info('Successfully created PostGIS store', stationDataStore);
+  } else {
+    console.error('Failed creating PostGIS store (maybe already existing ',
```

Comment:
closing bracket is missing

```suggestion
    console.error('Failed creating PostGIS store (maybe already existing) ',
```